picard/const/defaults.py

```diff
@@ -150,3 +150,6 @@
 DEFAULT_NAMING_PRESET_ID = "Preset 1"
 
 DEFAULT_TIME_FORMAT = '%Y-%m-%d %H:%M:%S'
+
+DEFAULT_COVER_MIN_SIZE = 250
+DEFAULT_COVER_MAX_SIZE = 1000
```

Comment:
1. I would prefer constants and variables like these to include the UoM in their name i.e. `DEFAULT_COVER_MIN_SIZE_PX`.
2. I wonder whether it is more normal for images to come in binary powers i.e. should these be 256 and 1024?